src/fosslight_reuse/_help.py

```diff
@@ -7,20 +7,26 @@
 _HELP_MESSAGE_REUSE = """
     Usage: fosslight_reuse [Mode] [option1] <arg1> [option2] <arg2>...
         ex) fosslight_reuse lint -p /home/test/ -f "notice/sample.py,src/init.py"
+            fosslight_reuse add -a -l MIT -c LGE
```

Comment:
-a ? <-- legacy option?

Reply:
-c LGE is working?

Reply:
Yes, -a is not working now and this example isn't correct. So, I'll update this example at next commit.